sqlmesh/core/console.py

```diff
@@ -208,7 +217,10 @@ def __init__(self, console: t.Optional[RichConsole] = None, **kwargs: t.Any) ->
         self.evaluation_model_progress: t.Optional[Progress] = None
         self.evaluation_model_tasks: t.Dict[str, TaskID] = {}
         self.evaluation_model_batches: t.Dict[Snapshot, int] = {}
+
+        # Put in temporary values that are replaced when evaluating
         self.evaluation_environment_naming_info = EnvironmentNamingInfo()
```

Comment:
just name this self.environment_naming_info

Reply:
It is only valid during evaluations which is why it has that name. Also this isn't a new variable I added it already exists I just moved it down.